template/packages/backend/src/app.module.ts

```diff
@@ -45,7 +43,6 @@ const config = new ConfigService()
       synchronize: config.isDevDatabase,
     }),
   ],
-  controllers: [AppController],
-  providers: [AppService, { provide: ConfigService, useValue: config }],
```

Comment:
Why do we no longer need the AppService and the AppController? Was this just a boilerplate module we are removing?

Reply:
AppController was an example of how to write a REST service. My thinking is if we're pushing GraphQL we don't need a REST example.